ddtrace/internal/telemetry/writer.py

```diff
@@ -382,10 +389,19 @@ def add_integration(self, integration_name, auto_enabled):
             "enabled": True,
             "auto_enabled": auto_enabled,
             "compatible": True,
-            "error": "",
+            "error": self._error[1],  # the integration error only takes a message, no code
```

Comment:
is this any error or the error specifically for this integration?

what if the last captured error is from the writer? it wouldn't make sense here

Reply:
Yup, I plan to change this we should pass the error directly to add_integrations